tests/lib_testslide.py

```diff
@@ -85,6 +85,20 @@ def gives_correct_error_message_for_invalid_types(self):
                 self.skip_first_arg, self.callable_template, (1, 2, 3, 4), {}
             )
 
+    @context.example("Invalid return Type raises TypeError")
+    def assert_raised_typeerror(self):
+        with self.assertRaises(
+            TypeError,
+                msg=(
+                "Call with incorrect return types at "
+                "/Users/jottosson/TestSlide/tests/sample_module.py:101:\n"
```

Comment:
Please don't use your own paths here, as it'll break on other machines. Check the `trim_path_prefix` parameter that comes from the CLI. You should use it on the code that generates the exception to trim the prefix.